### ISSUE TITLE
Add a way to know about the first peer estimate guess

### ISSUE THREAD

**Issue by satazor** [peer-base/peer-base#25]
Use-case: display a loading until the number of peers on a collaboration are being determined. We can do this by listening to the `membership changed` event but it's not triggered when we are the only peer in it.

It would be awesome to know when the first peer estimate was successful or failed to hide the loading an display the peers or an error respectively.

**Comment by pgte** [comment]
The peer count estimate is at the app level, not collaboration.
At the collaboration level, it's hard to know that we're alone for sure. In this case, I think that the best way is to timeout.
The maximum timeout while being alone can be calculated by:

`options.targetGlobalMembershipGossipFrequencyMS x 2 peers x 2 + pubsubLatencyMS`

The option `options.targetGlobalMembershipGossipFrequencyMS` defaults to 1000 milliseconds.

In the formula above, `pubsubLatencyMS` is not easily knowable, but you can put a ceiling on it.

@satazor does this make sense to you?

**Comment by satazor** [comment]
@pgte could you explain what the `2` in the formula are?

Regarding having to do this ourselves: could we have this event built-in in the collaboration somehow? Not sure about the naming though.

**Comment by pgte** [comment]
`options.targetGlobalMembershipGossipFrequencyMS` is a target average. The real value is `Math.random() * options.targetGlobalMembershipGossipFrequencyMS * 2`, hence the `2`.

Makes sense?

**Comment by satazor** [comment]
Understood.

> Regarding having to do this ourselves: could we have this event built-in in the collaboration somehow? I think this use-case will be very common in apps. Not sure about the naming though.

What do you think?

**Comment by pgte** [comment]
I agree. I'm thinking of an event named something like `probably alone`, that will be fired only once in the lifetime of this collaboration, once this timeout has occurred.

@satazor would this work for you?

**Comment by pgte** [comment]
I like awkward

**Comment by satazor** [comment]
Let's go with `probably alone` for now. Would you be able to do this before our demo? @VictorBjelkholm suggested to add a loading for the peers count, as I also suggested to you, but I need this feature to make it work.
